repository/src/main/java/org/alfresco/repo/search/impl/solr/SolrChildApplicationContextFactory.java

```diff
@@ -101,118 +114,96 @@ public boolean isUpdateable(String name)
                 && !name.equals(SolrChildApplicationContextFactory.ARCHIVE_MEMORY)
                 ;
     }
-
+    
+    private String getPropertyValue(JSONObject core, String name)
+    {
+        switch (name)
+        {
+            case SolrChildApplicationContextFactory.ALFRESCO_ACTIVE:
+            case SolrChildApplicationContextFactory.ARCHIVE_ACTIVE:
+                return core.getBoolean("Active") ? "true" : "false";

```

Comment:
Does it make sense to use static string variables for these?

Reply:
Refactored this Strings to Constants.